src/librustc_resolve/build_reduced_graph.rs

```diff
@@ -404,6 +404,29 @@ impl<'a, 'b:'a, 'tcx:'b> GraphBuilder<'a, 'b, 'tcx> {
             }
 
             ItemMod(..) => {
+                let child = parent.children.borrow().get(&name).cloned();
```

Comment:
Seems a shame to clone the child; I would have thought there was a way to structure this to avoid that.